airgun/views/hostgroup.py

```diff
@@ -118,9 +130,7 @@ class organizations(SatTab):
     @View.nested
     class activation_keys(SatTab):
         TAB_NAME = 'Activation Keys'
-        activation_keys = TextInput(
-            locator=".//foreman-react-component[contains(@data-props, 'kt_activation_keys')]"
-        )
+        activation_keys = ActivationKeyDropDown(locator=".//div[@class='rbt clearfix open']")
```

Comment:
Do you want to explicitly restrict the locator to these three classes, or the div that includes these three classes? We don't want to write locators that will be highly sensitive to page style changes.

Reply:
I will try to see if I can add more solid locator here.

Reply:
updated !